### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ascott1**
This offers general cleanup and better consistency to the two current guides as well as incorporate's @anselmbradford's feedback from #19.

## Review

Anyone from CFPB front-end! Let's get two thumbs up before merging.

### CODE REVIEW

File: javascript.md [hunk 1]
```diff
@@ -91,14 +73,15 @@ Where possible, improve readability by keeping lines below the 80-character mark
 - Functions, variables, methods, objects and instances should be named using `camelCase`.
 - Constructors and prototypes should use `UpperCamelCase`.
 - Symbolic constants should use `UPPERCASE`.
+- Encapsulated (private) variables and methods should use `_underscoreCamelCase`.
 
 ## Semicolons`;`
 
-The majority of JavaScript developers [prefer using semicolons][6] at the end of lines and we recommend following this practice. This choice is done to avoid potential issues with Automatic Semicolon Insertion _(ASI)_.
+TWe advocate for using semicolons at the end of each line. This choice is done to avoid potential issues with Automatic Semicolon Insertion _(ASI)_.
```
Comment (anselmbradford) [inline javascript.md]: `TWe` --> `We`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
 
 # CFPB front-end team
 
-> A repository for the discussion and publication of the CFPB front-end team standards.
-
-----
+A guide for front-end development at the [CFPB](http://cfpb.github.io/)
```
Comment (anselmbradford) [inline README.md]: Minor... could have a period at the end.

File: javascript.md [hunk 3]
```diff
@@ -91,14 +73,15 @@ Where possible, improve readability by keeping lines below the 80-character mark
...
 
 ## Strings
 
-Strings should always be quoted using the same quotation mark. Use `'` or `"` consistently throughout your codebase. Ensure the team is using the same quotation mark in every portion of JavaScript that's authored. In general, we prefer to use sing quotation parks `'`.
+Strings should always be quoted using the same quotation mark. Use `'` or `"` consistently throughout your codebase. Ensure the team is using the same quotation mark in every portion of JavaScript that's authored. In general, we prefer to use sing quotation marks `'`.
```
Comment (anselmbradford) [inline javascript.md]: `sing` --> `string`

**Comment by ascott1** [comment]
All of @anselmbradford's feedback has been addressed. Are we good to merge?

**Comment by anselmbradford** [comment]
SGTM

**Comment by ascott1** [comment]
:stuck_out_tongue_winking_eye:

**Comment by wpears** [comment]
Do it to it

**Comment by ascott1** [comment]
Merrrrge it @wpears or @anselmbradford

**Pull request merged**